source/plugin/commands/arguments.rst

```diff
@@ -174,25 +174,27 @@ The parser in this example reads two input arguments and converts them to a vect
 .. code-block:: java
 
    import com.flowpowered.math.vector.Vector2i;
-   import org.spongepowered.api.util.command.args.ArgumentParseException;
-   import org.spongepowered.api.util.command.args.CommandArgs;
+   import org.spongepowered.api.command.args.ArgumentParseException;
+   import org.spongepowered.api.command.args.CommandArgs;
    import org.spongepowered.api.text.Text;
-   import org.spongepowered.api.util.command.args.CommandElement;
+   import org.spongepowered.api.command.args.CommandElement;  
```

Comment:
**[Docs Guideline 14](https://docs.spongepowered.org/en/contributing/spongedocs.html)**: *Imports should be written out in code blocks the first time they are referenced in each article, but not repeated after the first time*.